app/api/session_types.py

```diff
@@ -85,6 +85,7 @@ class SessionTypeRelationshipRequired(ResourceRelationship):
     """
     SessionType Relationship
     """
+    methods = ['GET', 'PATCH']
```

Comment:
If only GET and PATCH are allowed, can you modify the decorators to also check only on PATCH instead of PATCH,DELETE?

Reply:
Resolved